core/src/modules/configLoaders/index.ts

```diff
@@ -322,5 +323,9 @@ export async function deleteLockedObjects(seenIds) {
   // back to normal
   deletedIds["app"] = await deleteApps(seenIds.app);
 
+  // Enqueue deletion tasks
+  const task: CLSTask = api.tasks.tasks["destroy"];
+  await task.runWithinTransaction({}, {});
+
```

Comment:
I'm guessing this is for the case where we are running via `grouparoo apply`?  In some of the other cases (`grouparoo start` and `grouparoo run`) the task will be handled for us, and in `grouparoo validate` we probably don't want to run this.

If this is added for the `grouparoo apply` case, perhaps we run it from within `src/bin/apply` to indicate that it's special for that command.

Reply:
Yeah it was for the apply case, but I just removed it entirely. Even if you run `apply` I don't think there's much of a benefit to enqueuing the tasks here. They won't actually be processed until you run the server, where in 30s when the recurring task runs they'll get enqueued anyway.

Reply:
It looks like the `task.runWithinTransaction()` is still here?

Reply:
derp, missed the commit for that. nice catch!
should be good now.